website/pages/en/index.js

```diff
@@ -74,115 +71,211 @@ HomeSplash.propTypes = {
   language: React.PropTypes.string
 };
 
+const TldrSection = () => (
+  <div className="tldrSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <div style={{ display: "flex", flexFlow: "row wrap" }}>
+        <div style={{ flex: "1 1 auto" }}>
+          <h2>What is Prettier?</h2>
+          <ul>
+            <li>An opinionated code formatter</li>
+            <li>Supports many languages</li>
+            <li>Integrates with the most editors</li>
+            <li>Has few options</li>
+          </ul>
+        </div>
+        <div style={{ flex: "1 1 auto" }}>
+          <h2>Why?</h2>
+          <ul>
+            <li>You press save and code is formatted</li>
+            <li>No need to discuss style in code review</li>
+            <li>Save you time and energy</li>
+          </ul>
+        </div>
+      </div>
+    </Container>
+  </div>
+);
+
+const LanguagesSection = () => (
+  <div
+    className="languagesSection productShowcaseSection paddingTop paddingBottom"
+    style={{ textAlign: "center" }}
+  >
+    <Container>
+      <h2 style={{ margin: 0 }}>Language Support</h2>
+      <GridBlock
+        align="center"
+        contents={siteConfig.supportedLanguages.map(language => ({
+          title: language.name,
+          image: language.image,
+          imageAlign: "top",
+          content: language.variants.join("\n\n")
+        }))}
+        layout="fourColumn"
+      />
+    </Container>
+  </div>
+);
+
+const Editor = ({ content = "", image, name }) => (
+  <div style={{ display: "flex", width: "235px", padding: "20px" }}>
+    <img src={image} style={{ width: "100px", height: "100px" }} />
+    <div style={{ flexGrow: 1, paddingLeft: "8px" }}>
+      <h3 style={{ marginBottom: "-16px" }}>{name}</h3>
+      <MarkdownBlock>{content}</MarkdownBlock>
+    </div>
+  </div>
+);
+
+Editor.propTypes = {
+  content: React.PropTypes.string,
+  image: React.PropTypes.string.isRequired,
+  name: React.PropTypes.string.isRequired
+};
+
+const EditorSupportSection = () => (
+  <div className="editorSupportSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <h2>Editor Support</h2>
+      <div
+        style={{
+          display: "flex",
+          flexFlow: "row wrap",
+          justifyContent: "space-around"
+        }}
+      >
+        {siteConfig.editors.map(editor => (
+          <Editor key={editor.name} {...editor} />
+        ))}
+      </div>
+    </Container>
+
+    <div style={{ float: "right" }}>
+      <span>Got more? </span>
+      <a
+        href={`${siteConfig.githubUrl}/edit/master/website/data/editors.yml`}
+        className="button"
+      >
+        Send a PR
+      </a>
+    </div>
+  </div>
+);
+
+const bash = string => `~~~bash\n${string}\n~~~`;
```

Comment:
How difficult would it be to make this a template tag? i.e.

```js
bash`
$ foo
bar
baz
`
```

Reply:
Easy enough - pushed a commit that does that, does it add any value?